compiler/lib/src/main/scala/codegen/CppWriter/CppDocCppWriter.scala

```diff
@@ -95,11 +95,10 @@ object CppDocCppWriter extends CppDocWriter {
         val retType = function.retType.getCppType match {
           case "" => ""
           case t => s"$t "
-
         }
         in.classNameList match {
-          case head :: _ => {
-            val line1 = line(s"$retType$head ::")
+          case _ :: _ => {
```

Comment:
I think we can change line 100 to `case _ => {`.

Reply:
You will also need to put the `Nil` case first to make this work. I think it's a little clearer than writing `_ :: _` to match against a list of size >= 1.

Reply:
However, I see that the `_ :: _` pattern is used elsewhere in the same file. So let's keep it!